website/src/pages/docs/grid.mdx

```diff
@@ -0,0 +1,64 @@
+---
+menu: Guides
+title: Grid
+order: 1
+---
+
+# Grid
+
+Quickly style your app with column and rows. 
+
+## Create a grid
+
+- Give the property `row` to a Box to automatically create a grid.
+- Give the property `col` to a Box to create a column.
+- Fill up the grid by giving a value comprised between 0 and 1 to each `col`.
+- By default, a `col` width is set to 1 and thus takes the entire space.
+- You may also use fractions. For example: 1/2 or 1/4. 
+- The last `col` will automatically fill up the remaining space. 
```

Comment:
Nop, the last `col` is not specific than any other col.

Reply:
Just in this example.

Reply:
Au dessus tu parles de généralités et là tout d'un coup tu parles de quelque chose de spécifique par rapport à l'exemple qui suit. On a du mal à comprendre.

Reply:
J'ai essayé de faire un exemple très général qui, selon moi,  suffisait à comprendre le fonctionnement global de la grille: "On crée un row comme ceci, une col comme cela, et voici comment ajuster leurs tailles respectives". 

Les explications sur la width et les fractions ne me paraissent pas spécifiques au code du dessous, elles sont au contraire valables dans tous les cas de figure. 

Si tu préfères exposer plusieurs cas précis au lieu d'un exemple général, quels exemples souhaites-tu que j'implémente?

Reply:
C'est ce que je dis rien n'est spécifique sauf `The last "col" will automatically fill up the remaining space.`

Tu dois séparer le générique et le spécifique.

Pour ce qui est de l'exemple, tu dois implémenter tous ceux là : https://smooth-ui-v10.smooth-code.com/docs-basics-grid

Mettre en place également des styles particulier pour que visuellement on voit les grilles.